src/Hosting.Services.Remoting/RemotingListenerBuilder.cs

```diff
@@ -29,6 +29,11 @@ public abstract class RemotingListenerBuilder<TService> : IListenerBuilder<TServ
 			string name,
 			FabricTransportRemotingListenerSettings? settings = null)
 		{
+			if (settings == null)
```

Comment:
From a quick look into SF code, passing settings from outside may interfere with secure remoting, we need to check that carefully.

Reply:
This is exactly what the SF docs say to do to enable data contract serialization - https://learn.microsoft.com/en-us/azure/service-fabric/service-fabric-reliable-services-exception-serialization

Reply:
Yes, but secure remoting depends on the same settings class and we are not populating the relevant fields. It might work, but I would triple check.